workshops/test/test_person.py

```diff
@@ -947,9 +951,12 @@ def test_all_requirements_satisfied(self):
                          set())
 
     def test_some_requirements_are_fulfilled(self):
+        # Homework was accepted, the second time.
```

Comment:
Yes, "SWC **demo**" should be missing -- it covers another case (when there are TrainingProgress records, but none of them have state == `passed`). Look at "SWC **Homework**" (lines 904-907) -- it does what I said in summary.